docs/language.md

```diff
@@ -188,7 +283,7 @@ Rule expressions are a subset of JavaScript expressions, and include:
 
 These global variables are available in expressions:
 
-    root - The root of your Firebase database.
+    root - The root location of a Firebase database.
     auth - The current auth state (if auth != null the user is authenticated auth.uid
```

Comment:
"The current auth state (if auth != null the user is authenticated auth.uid is their user identifier (a unique string value)."
>

"The current auth state (if auth != null, the user is authenticated, and auth.uid is their user identifier string)."